crates/gpui/src/text_system/line_wrapper.rs

```diff
@@ -225,13 +225,15 @@ mod tests {
                 font: font("Helvetica"),
                 color: Default::default(),
                 underline: Default::default(),
+                strikethrough: Default::default(),
```

Comment:
NIT but come on, `None` is even shorter to write, not speaking of reading 🙂 
There's a few more places that would benefit from this, but feel free to ignore this comment if you prefer the obscure syntax more.


```suggestion
                strikethrough: None,
```